LMS/Auth/Resources/views/components/footer.blade.php

```diff
@@ -1,36 +1,19 @@
 <footer class="footer">
     <div class="container">
-        <nav class="float-left">
+        <nav class="pull-left">
             <ul>
                 <li>
-                    <a href="https://www.creative-tim.com">
-                        Creative Tim
-                    </a>
-                </li>
-                <li>
-                    <a href="https://creative-tim.com/presentation">
-                        About Us
-                    </a>
-                </li>
-                <li>
-                    <a href="http://blog.creative-tim.com">
-                        Blog
-                    </a>
-                </li>
-                <li>
-                    <a href="https://www.creative-tim.com/license">
-                        Licenses
+                    <a href="http://www.creative-tim.com">
+                        {{ config('app.name') }}
                     </a>
                 </li>
             </ul>
         </nav>
-        <div class="copyright float-right">
+        <div class="copyright pull-right">
             ©
-            <script>
-                document.write(new Date().getFullYear())
-            </script>
-            2021, made with <i class="material-icons">favorite</i> by
-            <a href="https://www.creative-tim.com" target="_blank">Creative Tim</a> for a better web.
+            {{ config('app.name') }}
+            <script> document.write(new Date().getFullYear()) </script>
```

Comment:
Caso queira tirar esse javascript feioso, você pode renderizar o ano com blade! 
```suggestion
            {{ date('Y')}}
```

Reply:
Corrigido.